MapboxNavigation/RouteMapViewController.swift

```diff
@@ -128,13 +128,11 @@ class RouteMapViewController: UIViewController {
     func resumeNotifications() {
         NotificationCenter.default.addObserver(self, selector: #selector(willReroute(notification:)), name: RouteControllerWillReroute, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(didReroute(notification:)), name: RouteControllerDidReroute, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(didReroute(notification:)), name: RouteControllerDidFailToReroute, object: nil)
```

Comment:
`didReroute(_:)` currently calls `statusView.hide(delay:animated:)`, balancing out the call to `statusView.show(_:showSpinner:)` in `willReroute(_:)`. Does this mean the status view will show “Rerouting…” indefinitely?

Reply:
No, we hide it here: https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/RouteMapViewController.swift#L257

Now we're silently doing nothing when the reroute fails. Even in a case where we do fail to reroute, `RouteControllerDidReroute`  will always be called.

Reply:
Also, tested.

Reply:
But doesn’t this change remove the call to `didReroute(notification:)` that you linked to, or does that method still get called some other way?

Reply:
If rerouting failed, why does `RouteControllerDidReroute` get posted?